gsoc2018-bharat/src/MakeDictionary.py

```diff
@@ -0,0 +1,54 @@
+import sys
+import re
+import os
+import codecs
+
+def upcase_first_letter(s):
+	return s[0].upper() + s[1:]
+
+# Replaces the Gensim class LineSentence
+class MySentences(object):
```

Comment:
Better name for the corpus iterator ? WikiDescriptions or something like that ?

Reply:
Okay